codestodians/food-basket-front-end
# [Feature: #164501035] Refine profile page

#### What does this PR do?
- Adds a form and a picture holder to the profile page
- Enables update

#### Description of Task to be completed?
- Add update user mutation
- Add form for displaying user details
- Add validation

#### How should this be manually tested?
- Check out into the `feature/refine-profile-page-164501035` branch.
- Navigate to the profile page once logged in.
- Updating any of the user details should be successful if proper data is passed in.

#### Extra details
- The page still needs more work in terms of finesse
- Given the impending deprecation of redux, work should be done on apollo state persistence.
- We need a toaster/notification of some sort to indicate success or failure of certain actions.

#### What are the relevant pivotal tracker stories?
- [#164501035](https://www.pivotaltracker.com/story/show/164501035)
